bot/exts/evergreen/cowsay.py

```diff
@@ -0,0 +1,64 @@
+from random import choice
+
+from cowsay import get_output_string
+from discord import Embed
+from discord.ext import commands
+
+from bot.constants import Colours, NEGATIVE_REPLIES
+
+
+class Cowsay(commands.Cog):
+    """Cog for the cowsay command."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(aliases=["cow"])
+    async def cowsay(self, ctx: commands.Context, character: str = "Cow", *, text: str = "I'm a Cow") -> None:
+        """
+        Generates some cowsay ASCII art and sends it in Discord.
+
+        There are multiple character options to choose from. Here is a list:
+
+        Cow
+        Dragon
+        Cheese
+        Beavis
+        Daemon
+        Ghostbusters
+        Kitty
+        Meow
+        Milk
+        Pig
+        Stegosaurus
+        Stimpy
+        Turkey
+        Turtle
+        Tux
```

Comment:
Could we make them be separated by commas i.e. `,`.

The current help embed takes a lot of space.

Reply:
Another comment, rather than putting these characters manually, we would have to change the documentation whenever a new character is added. This wouldn't something nice.

A possible solution to this is to use `help=` parameter in `discord.ext.commands` and pass in the characters to it, using f-strings. The characters can be seen like this:
```py
cowsay.char_names
```

Reply:
Cowsay.char_names returns a dictkeys object, so we'd have to filter it before returning it or it looks bad

Reply:
`', '.join` exists.

Reply:
Resolved in https://github.com/python-discord/sir-lancebot/pull/699/commits/34183c6024654ab42e5fe61b1158faee361c834a